osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSubScreen.cs

```diff
@@ -430,10 +389,8 @@ protected override void Dispose(bool isDisposing)
                 client.RoomUpdated -= onRoomUpdated;
                 client.LoadRequested -= onLoadRequested;
             }
-        }
 
-        private class UserModSelectOverlay : LocalPlayerModSelectOverlay
-        {
+            modSettingChangeTracker?.Dispose();
```

Comment:
This is a drive-by fix for this not having been disposed before, right? Looks to be a good idea, just double-checking.

Reply:
is github playing tricks? i didn't add this line, it was already there.

Reply:
Interesting, seems like something went wrong there in the web diff. This does not appear as a change anymore.